spec/lib/geocoding_spec.rb

```diff
@@ -27,19 +28,38 @@
         expect(Geocoder).not_to receive(:coordinates)
         subject.coordinates
       end
+
+      it "does not trigger a Google Geocoding API hit event", :dfe_analytics do
```

Comment:
I'm wondering if the whole test shouldn't be :dfe_analytics enabled, so that we don't accidentally test for a negative when its switched off or typoed?

Reply:
Do you mean enabling analytics over all the tests in the file instead of on an individual basis? `RSpec.describe Geocoding, geocode: true, :dfe_analytics do`